extension/export_AdcMatrix.cc

```diff
@@ -63,13 +63,15 @@ void export_AdcMatrix(py::module& m) {
         .def_property_readonly("mospaces", &AdcMatrix::mospaces_ptr)
         .def_property_readonly("is_core_valence_separated",
                                &AdcMatrix::is_core_valence_separated)
-        .def("compute_apply", &AdcMatrix::compute_apply)
+        .def("compute_apply", &AdcMatrix::compute_apply, py::arg("block"), py::arg("in"),
```

Comment:
I think we should start using literal operators instead of `py::arg`, because it makes the code shorter:
```C++
.def("compute_apply", &AdcMatrix::compute_apply, "block"_a, "in"_a,
```

[Link](https://pybind11.readthedocs.io/en/stable/basics.html#keyword-arguments)

Note: needs a `using namespace pybind11::literals`.

Reply:
Good call.